tests/nest_compartmental_tests/resources/cm_default.nestml

```diff
@@ -15,11 +15,11 @@ neuron cm_default:
 
         ### ion channels ###
         # initial values state variables sodium channel
-        m_Na real = 0.0
-        h_Na real = 0.0
+        m_Na real = 0.01696863
```

Comment:
do these correspond with the default initial values in NEST?

Reply:
Yes, the change in NEST 3.6 is that these values are computed automatically based on the initialization of `v_comp`, so that these are the asymptotic equilibria of the state variables at that `v_comp` value. We don't have an appropriate way to generalize this to NESTML as not all asymptotic equilibria may be analytically computable.